src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -11261,99 +11266,103 @@ void Player_DetectSecrets(PlayState* play, Player* this) {
-    /*  5 */ PLAYER_CSMODE_56,
-    /*  6 */ PLAYER_CSMODE_8,
-    /*  7 */ PLAYER_CSMODE_0,
-    /*  8 */ PLAYER_CSMODE_0,
-    /*  9 */ PLAYER_CSMODE_135,
-    /* 10 */ PLAYER_CSMODE_21,
-    /* 11 */ PLAYER_CSMODE_61,
-    /* 12 */ PLAYER_CSMODE_62,
-    /* 13 */ PLAYER_CSMODE_60,
-    /* 14 */ PLAYER_CSMODE_63,
-    /* 15 */ PLAYER_CSMODE_64,
-    /* 16 */ PLAYER_CSMODE_65,
-    /* 17 */ PLAYER_CSMODE_66,
-    /* 18 */ PLAYER_CSMODE_70,
-    /* 19 */ PLAYER_CSMODE_19,
-    /* 20 */ PLAYER_CSMODE_71,
-    /* 21 */ PLAYER_CSMODE_72,
-    /* 22 */ PLAYER_CSMODE_67,
-    /* 23 */ PLAYER_CSMODE_73,
-    /* 24 */ PLAYER_CSMODE_74,
-    /* 25 */ PLAYER_CSMODE_75,
-    /* 26 */ PLAYER_CSMODE_68,
-    /* 27 */ PLAYER_CSMODE_69,
-    /* 28 */ PLAYER_CSMODE_76,
-    /* 29 */ PLAYER_CSMODE_116,
-    /* 30 */ PLAYER_CSMODE_0,
-    /* 31 */ PLAYER_CSMODE_40,
-    /* 32 */ PLAYER_CSMODE_0,
-    /* 33 */ -PLAYER_CSMODE_52, // the only negative one
-    /* 34 */ PLAYER_CSMODE_42,
-    /* 35 */ PLAYER_CSMODE_43,
-    /* 36 */ PLAYER_CSMODE_57,
-    /* 37 */ PLAYER_CSMODE_81,
-    /* 38 */ PLAYER_CSMODE_41,
-    /* 39 */ PLAYER_CSMODE_53,
-    /* 40 */ PLAYER_CSMODE_54,
-    /* 41 */ PLAYER_CSMODE_44,
-    /* 42 */ PLAYER_CSMODE_55,
-    /* 43 */ PLAYER_CSMODE_45,
-    /* 44 */ PLAYER_CSMODE_46,
-    /* 45 */ PLAYER_CSMODE_47,
-    /* 46 */ PLAYER_CSMODE_48,
-    /* 47 */ PLAYER_CSMODE_49,
-    /* 48 */ PLAYER_CSMODE_50,
-    /* 49 */ PLAYER_CSMODE_51,
-    /* 50 */ PLAYER_CSMODE_77,
-    /* 51 */ PLAYER_CSMODE_78,
-    /* 52 */ PLAYER_CSMODE_79,
-    /* 53 */ PLAYER_CSMODE_80,
-    /* 54 */ PLAYER_CSMODE_81,
-    /* 55 */ PLAYER_CSMODE_82,
-    /* 56 */ PLAYER_CSMODE_83,
-    /* 57 */ PLAYER_CSMODE_84,
-    /* 58 */ PLAYER_CSMODE_85,
-    /* 59 */ PLAYER_CSMODE_86,
-    /* 60 */ PLAYER_CSMODE_87,
-    /* 61 */ PLAYER_CSMODE_88,
-    /* 62 */ PLAYER_CSMODE_89,
-    /* 63 */ PLAYER_CSMODE_90,
-    /* 64 */ PLAYER_CSMODE_91,
-    /* 65 */ PLAYER_CSMODE_92,
-    /* 66 */ PLAYER_CSMODE_94,
-    /* 67 */ PLAYER_CSMODE_95,
-    /* 68 */ PLAYER_CSMODE_100,
-    /* 69 */ PLAYER_CSMODE_101,
-    /* 70 */ PLAYER_CSMODE_98,
-    /* 71 */ PLAYER_CSMODE_99,
-    /* 72 */ PLAYER_CSMODE_102,
-    /* 73 */ PLAYER_CSMODE_103,
-    /* 74 */ PLAYER_CSMODE_104,
-    /* 75 */ PLAYER_CSMODE_112,
-    /* 76 */ PLAYER_CSMODE_113,
-    /* 77 */ PLAYER_CSMODE_117,
-    /* 78 */ PLAYER_CSMODE_104,
-    /* 79 */ PLAYER_CSMODE_104,
-    /* 80 */ PLAYER_CSMODE_105,
-    /* 81 */ PLAYER_CSMODE_106,
-    /* 82 */ PLAYER_CSMODE_107,
-    /* 83 */ PLAYER_CSMODE_108,
-    /* 84 */ PLAYER_CSMODE_109,
-    /* 85 */ PLAYER_CSMODE_110,
-    /* 86 */ PLAYER_CSMODE_118,
-    /* 87 */ PLAYER_CSMODE_119,
-    /* 88 */ PLAYER_CSMODE_120,
-    /* 89 */ PLAYER_CSMODE_114,
-    /* 90 */ PLAYER_CSMODE_111,
-    /* 91 */ PLAYER_CSMODE_122,
+// Making a player csMode negative will behave as its positive counterpart
+// except will disable setting the start position
+#define DISABLE_PLAYER_CSMODE_START_POS -1
```

Comment:
Hmm, I know the reason why you make this just a define and multiply the element in the array (to use it in the comparison check). I'm not 100% sure how I feel about it. I wonder if this would be better as a macro like 
`DISABLE_PLAYER_CSMODE_START_POS(x) -(x)`
than still just compare `>= NONE` later, perhaps with a comment?

Reply:
A macro probably makes more sense here, I'm fine with doing it. Might reorder the name to:
`PLAYER_DISABLE_CSMODE_START_POS(x) -(x)`

Reply:
Oh wait, I named it that way to align with `gDisablePlayerCsModeStartPos` since they do the same thing